### ISSUE TITLE
player.name global sync setting

### ISSUE THREAD

**Issue by Lekenthu** [altmp/altv-issues#2207]
### Description of the problem

Since the last alt:V patch, the name property is now only synchronized to other players within the streaming distance. 
It would be nice if it could be set in server configuration to continue to synchronize with everyone. 

But why is this necessary?
For example, for scoreboard (online player list).

### Desired solution for the problem

-

**Comment by xxshady** [comment]
do you have `sendNames` option in your server config?

**Comment by xxshady** [comment]
or do you use .sendNames anywhere in your code?

**Comment by Lekenthu** [comment]
> do you have `sendPlayerNames` option in your server config?

Yes, it is currently at true value. I don't use sendNames anywhere.

**Comment by xxshady** [comment]
It's bug then

**Comment by FabianTerhorst** [comment]
Use synced meta data for global name sync. This was a breaking change that wasn't written in the changelog
